README.md

```diff
@@ -20,21 +20,21 @@ Given that we're considering fairness, the base of the toolbox is the triplet {x
 
 All methods must assume S and Y are multi-class.
 
-We use a dictionary to contain the triplet of type Dict\[str, pandas.DataFrame]
+We use a named tuple to contain the triplet
 
-    triplet = { 'x': dataframe, 's': dataframe, 'y': dataframe }
+    triplet = DataTuple(x=dataframe, s=dataframe, y=dataframe)
 
 The dataframe may be a little innefficient, but given the amount of splicing on conditions that we're doing it feels worth it.
 
 ## Separation of Methods
 
 We purposefully keep pre, during and post algorithm methods separate. This is because they have different return types.
 
-    pre-algorithm.run(train, test) -> Tuple[pandas.DataFrame, pandas.DataFrame]
-    in-algorithm.run(train, test) -> pandas.DataFrame
-    post-algorithm.run(preds, test) -> pandas.DataFrame
+    pre-algorithm.run(train: DataTuple, test: DataTuple) -> Tuple[pandas.DataFrame, pandas.DataFrame]
```

Comment:
This is completely correct at the moment, but I've been thinking about pre-algorithm returning `Tuple[DataTuple, DataTuple]`. It'll be a future PR, but seeing this has reminded me